atom/browser/ui/cocoa/atom_menu_controller.mm

```diff
@@ -223,6 +233,8 @@ - (void)addItemToMenu:(NSMenu*)menu
       [NSApp setWindowsMenu:submenu];
     else if (role == base::ASCIIToUTF16("help"))
       [NSApp setHelpMenu:submenu];
+    // left for backwards compatibility
+    // TODO(codebytere): remove in 5.0
```

Comment:
Contents of the `master` branch is supposed to be released in version 5, why not to remove everything right now?